spec/appendix.md

```diff
@@ -1,12 +1,57 @@
 ## Appendix
 
+### Trust Over IP Glossary - Controlled Terms 
+
+NOTE: 
+* The following set of identifiers are from the current [ToIP Glossary](https://docs.google.com/document/d/1fZByfuSOwszDRkE7ARQLeElSYmVznoOyJK4sxRvJpyM/edit?usp=sharing) (Google Doc). These definitions need to be moved over to the (under development) ToIP Terms tool (aka TEv2).
+* not all ToIP definitions are fully linked here. This whole section is more of an exemplar as of the time of writing.
+
+[[def: autonomous identifier]] 
```

Comment:
Do you think it would be better to include this term in the list of terms for autonomic identifier (it includes AID, etc) in order to leverage that hover-over text and links.